### ISSUE TITLE
[FEATURE REQUEST] account swaping on empties

### ISSUE THREAD

**Issue by SkOODaT** [PokemonGoMap/PokemonGo-Map#1332]
be awesome to have accounts swap out on say 10 empty requests for accounts that are soft banned and say maby after it switches if it still pulls empty requests everything sleeps to avoid spamming on ip softbans

**Comment by aduljr** [comment]
it does do that already essentially you give it a csv list of accounts, tell how many accounts you can use, how long to use those accounts and it pulls from the accounts waiting in queue. It will also do that for accounts that fail for one reason or another. Accounts that get pulled can be set to rest before going back into queue.

**Comment by SkOODaT** [comment]
i just want it to swap out soft banned accounts also say 10-15 empty requests and it swaps to another account like it does now with failed accounts if say all the accounts are posting empty responses the entire thing should go into a standby mode so its not spamming empty requests if your IP softbanned

**Comment by Alzurana** [comment]
Lemme add a little feature description:
Putting accounts on hold not only after 5 consecutive failures but also after 5-10 consecutive empty responses. (maybe configurable)
This gives them the time to recover from a soft ban and prevents soft banned accounts from returning false empty responses.

**Comment by Decepticon01** [comment]
Definitely interested in this feature.

**Comment by SkOODaT** [comment]
@Decepticon01 alot of people are going to be now id love to see someone finally do this :)
